pantheon-systems/drops-8
# Update to Drupal 8.0.0.

For more information, see https://www.drupal.org/node/2619030

Instructions to test:

- Create a new Drupal 8 site on Pantheon.
- When site creation is finished, visit dashboard.
- Switch to "git" mode.
- Clone your site locally.
- Apply the files from this PR on top of your local checkout.
  - git remote add drops-8 git@github.com:pantheon-systems/drops-8.git
  - git fetch drops-8
  - git merge drops-8/update-8.0.0
- Push your files back up to Pantheon.
- Switch back to sftp mode.
- Visit your site. Step through the installation process. Look for any problems.

Note that the redirection to core/installer.php is now done by a new, cleaner platform feature that is [being deployed to the platform right now](https://github.com/pantheon-cookbooks/endpoint/pull/132). If you try to test this PR before this feature has converged on your instance, you will get a WSOD with a database error. Do not be alarmed; proceed to the installer by manually adding `/core/installer.php` to the URL, and continue with the test.

- Test standard and minimal installation profiles
- Test an English and non-English install
- Insure the update.php will run through the browser (manually visit /update.php)
- Insure that you can install modules through the admin interface
- Insure that 'is-active' is added where appropriate (active link text is black instead of blue when you move the Main Navigation menu to a sidebar)
- Do other general ad-hoc tests to suit, to make sure everything is working well
